<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {

	background-image: url(:/newPrefix/bg.svg);
}

#groupe {

	background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0 rgba(87, 0, 155, 43), stop:1 rgba(198, 71, 255, 255));
border-radius : 18px;
border:0px;
}
#course_2{
	background-color: rgb(240, 240, 240);
	color: rgb(107, 29, 163);
border:0px;
border-radius: 20px;
font: bold 18px; }

#course_3{
	background-color: rgb(240, 240, 240);
	color: rgb(107, 29, 163);
border:0px;
border-radius: 20px;
font: bold 18px; }
QGroupBox {
	background-color: rgb(107, 29, 163);
border-radius : 18px;
border:0px;
}
#text {

background-color: rgba(0,0,0,0%)
}
#groupie {
border:0px;
}
QPushButton {
	color: rgb(255, 255, 255);
	
	background-color: rgb(90, 24, 136);
	border: 0px solid;
	border-radius : 16px;
	font: bold 18px;
}

QStackedWidget {
	
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 42), stop:1 rgba(219, 219, 219, 12));
}
QLabel{
	background-color: rgb(240, 240, 240);
color:white;
border:0px;
border-radius: 20px;
font: bold 18px;
}
QPushButton:hover {	
	background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:1, stop:0 rgba(161, 3, 212, 76), stop:1 rgba(113, 51, 70, 0));
	color: rgb(255, 255, 255);
}

#list {
	background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:1, stop:0 rgba(161, 3, 212, 76), stop:1 rgba(113, 51, 70, 0));
	margin-bottom: 20px;
	margin-left: 20px;
	margin-right: 20px;
	border-radius: 15px;
	gridline-color: rgba(47, 56, 67, 150);
	font-size: bold 20px;

}

#list::item {
	background-color: rgb(90, 24, 136);
	color: white;
	padding-left: 20px;
	padding-right: 20px;
font:20px;
}

#queuenumber{
	background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:1, stop:0 rgba(176, 0, 255, 93), stop:1 rgba(113, 51, 70, 0));
font: bold 45px;
color:white;
}</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>931</width>
     <height>601</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#gp {

	background-color: rgb(56, 0, 84);
}</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="home">
    <widget class="QGroupBox" name="groupie2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>441</width>
       <height>431</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="queuenumber">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>50</y>
        <width>181</width>
        <height>161</height>
       </rect>
      </property>
      <property name="text">
       <string>01</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="enrollee_name">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>160</y>
        <width>181</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(236, 93, 246);</string>
      </property>
      <property name="text">
       <string>Mark Mcphee</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="course">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>230</y>
        <width>391</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(236, 93, 246);</string>
      </property>
      <property name="text">
       <string>Information Technology</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="next_buttn">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>330</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>161</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0,0,0,0%)</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>undraw_Queue_j6ij.svg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <zorder>label</zorder>
     <zorder>queuenumber</zorder>
     <zorder>enrollee_name</zorder>
     <zorder>course</zorder>
     <zorder>next_buttn</zorder>
    </widget>
    <widget class="QLabel" name="course_2">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>460</y>
       <width>271</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Waiting on Queue</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupe">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>50</y>
       <width>401</width>
       <height>431</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QListWidget" name="list">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>10</y>
        <width>421</width>
        <height>441</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Justin Bieber</string>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QLabel" name="course_3">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>460</y>
       <width>271</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#course_2{
	background-color: rgb(240, 240, 240);
	color: rgb(107, 29, 163);
border:0px;
border-radius: 20px;
font: bold 18px; }</string>
     </property>
     <property name="text">
      <string>Next Enrollee</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <zorder>groupe</zorder>
    <zorder>groupie2</zorder>
    <zorder>course_2</zorder>
    <zorder>course_3</zorder>
   </widget>
   <widget class="QWidget" name="logout">
    <widget class="QPushButton" name="yes_buttn">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>330</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>YES</string>
     </property>
    </widget>
    <widget class="QPushButton" name="no_buttn">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>330</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>NO</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="gp">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>90</y>
       <width>431</width>
       <height>351</height>
      </rect>
     </property>
     <property name="title">
      <string>GroupBox</string>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>160</y>
        <width>291</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Nirmala UI</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
background-color: rgba(0,0,0,0%);
color:white;
}
</string>
      </property>
      <property name="text">
       <string>Are you sure you want to logout?</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>40</y>
        <width>201</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Nirmala UI</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
background-color: rgba(0,0,0,0%);
color:white;
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>question (1).svg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <zorder>gp</zorder>
    <zorder>yes_buttn</zorder>
    <zorder>no_buttn</zorder>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupie">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>81</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="appoint_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Student_new/img/logout.svg</normaloff>../Student_new/img/logout.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../facultydashboard_new/img/home.svg</normaloff>../facultydashboard_new/img/home.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <zorder>pushButton</zorder>
   <zorder>appoint_2</zorder>
  </widget>
 </widget>
 <resources>
  <include location="a.qrc"/>
 </resources>
 <connections/>
</ui>
